### PR TITLE
Feature/tooltips

### PR DESCRIPTION
**Pull request by starsinmypockets**

### CODE REVIEW

File: ckanext/dcat_usmetadata/templates/new-metadata.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div id="root" data-apiUrl="{{h.full_current_url()}}/../../api/3/action/package_create" data-apiKey="{{c.userobj.apikey}}" data-ownerOrg=123" ></div>
 <link rel="stylesheet" href="/css/main.245a4124.chunk.css" type"text/css">
-<script src="/js/2.d1ac1708.chunk.js"></script>
-<script src="/js/main.6437f873.chunk.js"></script>
+<script src="/js/2.5ed9dd22.chunk.js"></script>
+<script src="/js/main.55fdb757.chunk.js"></script>
```
Comment (adborden) [inline ckanext/dcat_usmetadata/templates/new-metadata.html:4]: Are these manually updated?

File: cypress/integration/app.spec.js [hunk 2]
```diff
@@ -6,4 +6,23 @@ describe('DCAT Metadata App: New loads', () => {
...
+
+  it('Select with optional fields work', () => {
+    cy.get('select[name=license]').select('MIT')
+    cy.get('input[name=license_others]').should('be.disabled')
+    cy.get('select[name=license]').select('Others')
+    cy.get('input[name=license_others]').should('be.enabled')
+  })
+  
+  it('Submit Required Metadata works', () => {
+    cy.requiredMetadata()
```
Comment (adborden) [inline cypress/integration/app.spec.js:25]: Maybe these should include a verb. It's not clear from reading what this is doing. E.g. input required metadata()

File: ckanext/dcat_usmetadata/templates/new-metadata.html (hunk 1 above)
Comment (starsinmypockets) [reply ckanext/dcat_usmetadata/templates/new-metadata.html:4]: Yes, atm -- would love to find a better suggestion

File: cypress/integration/app.spec.js (hunk 2 above)
Comment (starsinmypockets) [reply cypress/integration/app.spec.js:25]: "Submit" is a verb? Can you give an example?

File: ckanext/dcat_usmetadata/templates/new-metadata.html (hunk 1 above)
Comment (starsinmypockets) [reply ckanext/dcat_usmetadata/templates/new-metadata.html:4]: @adborden It's possible to remove the hash and pull down the static files from npm. The downside is that browsers are likely to cache the resources, interfering with updates.

**Pull request merged**